idf/overview.rst

```diff
@@ -0,0 +1,74 @@
+##############################
+Interim Data Facility Overview
+##############################
+
+Pre-operations activities may be carried out on Interim Data Facility (IDF) hosted on `Google Cloud Platform <https://cloud.google.com/>`_.
+The `Google Cloud documentation site <https://cloud.google.com/docs/overview>`_ provides an overview of its products and services.
+
+IDF is considered a production environment to support Data Preview releases.
+Development and Commissioning work continue on `LDF <../services/lsst-devl.rst>`_.
+
+
+Login credentials
+=================
+
+Rubin staff working on deploying and operating Data Production services receives a ``username@lsst.cloud`` identity.
+This IDF cloud identity is not an email and is independent from other Rubin accounts.
+Not everybody in the Data Production department needs this login; contact your pre-operations team leads if in doubt.
+
+2FA is required for the IDF identity.
+See :ref:`idf-internal-support` for lost password or login issues.
+
+
+Environments
```

Comment:
Why "Environments" and not "Projects"?

Reply:
I wanted something that's broader than the GCP "projects", as part of this session is about using Terraform in idf_deploy. But maybe this session can be split into two sessions?